### DISCUSSION TITLE
dockerless setup

### DISCUSSION THREAD

**Discussion by moongazer07** [dani-garcia/vaultwarden#3562] [Q&A]
I need a dockerless setup so I can deploy it to my website

I have latest php installation that my
direct admin panel offers

I have installed other js libs using softacolous

but I don't have docker on there
there is no way to install
and just having docker builds is not fair
to people like me

**Comment by RealOrangeOne** [comment]
Vaultwarden isn't written in PHP, so that's not useful.

You can download the pre-compiled binaries in the releases. Just set that running (will depend on your server host, but it's probably with a systemd service), and then make sure you can get traffic to it.

Without more details of your server setup, it's impossible to answer this properly. There are some [deployment examples](https://github.com/dani-garcia/vaultwarden/wiki/Deployment-examples) in the wiki which may help.
